Checkout v201 docs (#398)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/checkout/tutorials/address-integration.mdx

The following steps describe how to integrate the Google Address Validation API with the Commerce boilerplate template. You can use these steps as a reference to integrate other API providers as well.

<Aside type="note">

> The _**commerce-checkout.js**_ block is the only one that is fully functional and up-to-date with the latest version of the checkout drop-in component.

Reviewer: ```suggestion
The `commerce-checkout.js` block is the only fully functional implementation that's up-to-date with the latest checkout drop-in component.
```

> Use the following guidelines just as a reference when creating a new checkout experience.

Reviewer: ```suggestion
Use these steps as a reference when creating your own checkout experience.
```
